shared/common-adapters/web-view.desktop.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import * as React from 'react'
+import type {WebViewProps} from './web-view'
+
+class WebView extends React.PureComponent<WebViewProps> {
+  webviewRef: any
+
+  constructor(props: WebViewProps) {
+    super(props)
+    this.webviewRef = React.createRef()
+  }
+  componentDidMount() {
+    const css = (this.props.injections && this.props.injections.css) || ''
+    const javaScript = (this.props.injections && this.props.injections.javaScript) || ''
+    if (css || javaScript) {
+      this.webviewRef.current.addEventListener('dom-ready', () => {
```

Comment:
pretty sure you'll need to test `this.webviewRef.current`, especially if you type `webviewRef`